### ISSUE TITLE
Hide initial dots when running Nim compiler if --verbosity:0

### ISSUE THREAD

**Issue by jfilby** [nim-lang/Nim#18943]
Nim's compiler should give minimal output when --verbosity:0 is specified. However the first line outputs a series of dots as initial processing occurs.

Windows console is very slow to print any output, so I'd prefer if this was hidden for faster compile times (it adds up over the day). This is more noticeable on larger projects with many files, and hence more dots.

### Example
```
nim --verbosity:0 c test.nim
```

### Current Output
```
...................................
```

### Expected Output
No output.

### Possible Solution

* In file compiler\msgs.nim in the liMessage() procedure there's a call to: msgWrite(conf, ".")
* This could be where the dots are printed, but they shouldn't be for setting --verbosity:0.

### Additional Information

```
>nim -v
Nim Compiler Version 1.4.8 [Windows: amd64]
Compiled at 2021-05-25
```

**Comment by kaushalmodi** [comment]
@jfilby I cannot reproduce this issue in nim 1.6.0 or nim devel. Can you update to the latest nim release and confirm?

**Comment by jfilby** [comment]
This is resolved in Nim 1.6.0, thanks!
